lib/music/MusicManager.js

```diff
@@ -0,0 +1,34 @@
+const { Guild, Collection } = require("discord.js");
```

Comment:
Useless collection, MusicInterfactes should be added thru a structure.extend and a property